README.md

```diff
@@ -2,28 +2,19 @@
 
 A command-line tool for managing (create/update) EasyDNS DNS records using easyDNS rest API.
 
-```
-❯❯❯ python easydns-restapi-cli.py --help                   
-Usage: easydns-restapi-cli.py [options]
+![Sample Screenshot](./docs/example.png)
 
-Options:
-  --version             show program's version number and exit
-  -h, --help            show this help message and exit
-  -f CONF, --file=CONF  configuration file containing easyDNS API details
-  -c, --create          create new record
-  -u, --update          update existing record to new IP address
-  -H HOSTNAME, --hostname=HOSTNAME
-                        specify short hostname without domain name part. e.g:
-                        www for www.example.com
-  -a IPADDR, --address=IPADDR
-                        specify IP address for the hostname
-```
+### Requirements
```

Comment:
Let's add EasyDNS API in the requirement section.

Reply:
will do it , Nice